.github/workflows/push.yml

```diff
@@ -32,3 +32,14 @@ jobs:
         username: ${{ secrets.DOCKER_USERNAME }}
         password: ${{ secrets.DOCKER_PASSWORD }}
         buildargs: BINARY_VERSION
+    - name: Build Linting Image
```

Comment:
Should be moved to its own job, we'll need to figure out if we really need/want to build linting images using Commodore version tags, or if just having commodore-lint:latest is enough.

This really could be a separate action.

Reply:
No we only need latest. At least for now. I limited to push to master branch.